docs/source/deployment/prefect.md

```diff
@@ -1,245 +1,264 @@
 # Prefect
 
-This page explains how to run your Kedro pipeline using [Prefect Core](https://www.prefect.io/products/core/), an open-source workflow management system.
+This page explains how to run your Kedro pipeline using [Prefect 2.0](https://www.prefect.io/products/core/), an open-source workflow management system.
 
-In scope of this deployment, we are interested in [Prefect Server](https://docs.prefect.io/orchestration/server/overview.html#what-is-prefect-server), an open-source backend that makes it easy to monitor and execute your Prefect flows and automatically extends the Prefect Core. Prefect Server ships out-of-the-box with a fully featured user interface.
+In scope of this deployment, we are interested in a self hosted [Prefect Server](https://docs.prefect.io/2.10.17/host/), an open-source backend that makes it easy to monitor and execute your Prefect flows and automatically extends Prefect 2.0. We will use an [Agent that dequeues submitted flow runs from a Work Queue](https://docs.prefect.io/2.10.17/tutorial/deployments/#why-workpools-and-workers).
 
 ```{note}
-This deployment has been tested using kedro 0.17.6, 0.17.7 and 0.18.2 with prefect version 1.1.0.
-
-The current implementation has not been tested with prefect 2.0.0.
+This deployment has been tested using Kedro 0.18.10 with Prefect version 2.10.17. If you want to [make a deployment with Prefect 1.0](https://docs.kedro.org/en/0.18.10/deployment/prefect.html) you may want to check older versions.
 ```
 
 ## Prerequisites
 
-To use Prefect Core and Prefect Server, ensure you have the following prerequisites in place:
+To use Prefect 2.0 and Prefect Server, ensure you have the following prerequisites in place:
+
+- [Prefect 2.0 is installed](https://docs.prefect.io/2.10.17/getting-started/installation/#installing-the-latest-version) on your machine
+
+## Setup
+
+Configure your `PREFECT_API_URL` to point to your local Prefect instance:
+
+```bash
+prefect config set PREFECT_API_URL="http://127.0.0.1:4200/api"
+```
+
+Create a `.telemetry` file in the **root of your Kedro project** with the following content:
+
+```text
+consent: false
+```
```

Comment:
I don't know how I feel telling our users to opt-out telemetry like this 😬 but asking them to write `consent: true` feels a bit forceful. Maybe we should link somewhere else in the docs that explains what to do with the telemetry? cc @stichbury